### ISSUE TITLE
save event does not distinguish between motion and snapshot events

### ISSUE THREAD

**Issue by worikgh** [Motion-Project/motion#953]
Using motion 4.4.1 form debian archives on a Raspberry PI Zero

tl;dr I need different `save` events for snapshots.  A new `event`: `snapshot_save`

I am writing software that processes motion events.  The set up I have is taking snapshots every ten minutes, but  I only care about the events triggered by motion.  (Need the snapshots for a different reason)
I am having to filter out snapshot events by looking at he file name.
Hence I wish that the `save` event  for a event triggered by changing pixels was different for saving a snapshot and an event triggered by motion.

Am I missing something?  
If it is a good idea I could see myself putting the work in to do a pull request.
Byt I fear I am missing some subtlety...

**Comment by tosiara** [comment]
https://github.com/Motion-Project/motion/issues/913

**Comment by worikgh** [comment]
> #913

Not really.  Mine is a different issue

I have no trouble triggering a event when a snapshot is saved, I want a *different* event from saving a image triggered by detected motion

**Comment by tosiara** [comment]
`on_picture_save` called when snapshot is being saved. That is exactly what you asked based on your description. But the same script will be called if motion saves picture as part of the event, and if you manually trigger a snapshot. You will have to parse file name in order to be sure which event you got. If the file name contains format for snapshot - then it snapshot triggered the script. If the file name contains format for regular picrute save - then it was pasrt of motion event.
And it is exactly what was asked in the bug 913

**Comment by worikgh** [comment]
\pedantic{on}
I said `Hence I wish that the save event for a event triggered by changing pixels was different for saving a snapshot and an event triggered by motion.` 
Bad grammar, I know, but I thought I as clear.
\pedantic{off}

Is there a good reason not to add a event for snapshots?
